src/main.cpp

```diff
@@ -3,15 +3,16 @@
 #include "PlayerManager.hpp"
 #include "ChatManager.hpp"
 #include "ItemManager.hpp"
+#include "MissionManager.hpp"
 #include "NanoManager.hpp"
 #include "NPCManager.hpp"
 
 #include "settings.hpp"
 
 #if defined(__MINGW32__) && !defined(_GLIBCXX_HAS_GTHREADS)
-    #include "mingw/mingw.thread.h"
```

Comment:
This formatting is deliberate; for readability. The rest of the formatting changes in this file are ok, though.